### DISCUSSION TITLE
helm chart for Mirth-connect

### DISCUSSION THREAD

**Discussion by ewa-czezyk** [nextgenhealthcare/connect#6299] [Q&A]
Hello. Im trying to install mirth connect on k0s via helm. I have found only one repository: https://github.com/hiddehs/nextgen-connect-chart, but everything there, as well as templates, are very old. Has someone information, about newer version of helm chart?

**Comment by jonbartels** [comment]
That repo looks like a good start. I see some small mistakes but it looks OK to me. It seems similar to the one a coworker wrote.
